.github/workflows/update.yml

```diff
@@ -9,17 +9,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Fetch the repository code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           token: ${{ secrets.BOT_ACCESS_TOKEN }}
       - name: Fetch the schemes repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
-          repository: tinted-theming/base16-schemes
+          repository: tinted-theming/schemes
           path: schemes
           token: ${{ secrets.BOT_ACCESS_TOKEN }}
       - name: Update schemes
         uses: tinted-theming/base16-builder-go@latest
+        with:
+          path: schemes/base16
```

Comment:
As mentioned in description, maybe we don’t need this and we actually can remove fetching the `base16-schemes` repository? I very well could be wrong, but that was my understanding looking at the logic.

Reply:
Yeah, I believe we could drop the checkout of the schemes repo and rely on the builder, because by default it fetches the latest schemes from GitHub.

That’s how we currently do it for Emacs.

https://github.com/tinted-theming/base16-emacs/blob/b27ec32b2be7112cbec1b4c07c6a2cd40d38c85e/.github/workflows/update.yml

Reply:
I can make that change then. I think the documentation updates are still worthwhile to get it. I'll do this later today. Thanks for confirmation.

Reply:
I made this change and updated the PR description for merge commit message.